agent-core/src/main/java/com/newrelic/agent/android/util/Constants.java

```diff
@@ -12,6 +12,30 @@ public final class Transactions {
         public static final String CONTENT_TYPE = "content_type";
     }
 
+    public static final class ApolloGraphQLHeader {
+        public static final String OPERATION_NAME = "X-APOLLO-OPERATION-NAME";
+        public static final String OPERATION_TYPE = "X-APOLLO-OPERATION-TYPE";
+        public static final String OPERATION_ID = "X-APOLLO-OPERATION-ID";
+
+        public static final String OPERATION_NORMALIZE_NAME = "operationName";
+        public static final String OPERATION_NORMALIZE_TYPE = "operationType";
+        public static final String OPERATION_NORMALIZE_ID = "operationId";
+
+        public static String normalizeApolloHeader(String s) {
```

Comment:
I get your intention, but no code with the constants, please.

Reply:
This method seems general enough to avoid an `Apollo` reference. Should probably be a method in `HttpHeaders`.

Reply:
It's petty, but the operation s really more a translation than a normalizaton.

Reply:
got this, will do necessary changes.